### PR TITLE
Adding shadcn guide to README.md

### PR DESCRIPTION
**Pull request by Jonghakseo**
> `*` Please fill in the required items.

## Priority*

- [ ] High: This PR needs to be merged first for other tasks.
- [x] Middle: This PR should be merged quickly to prevent conflicts due to common changes. (default)
- [ ] Low: This PR does not affect other tasks, so it can be merged later.

## Purpose of the PR*

Add the SHADCN guide that`@rutamstwt` posted to the Discord channel to the README of the UI package.

## Changes*

I tested the guide he posted at a practical level and made any adjustments that needed to be made.

Added the tsc-alias course to the build so that the aliasing entered in tsconfig works properly.

## How to check the feature

Follow the guide to create and use a Button component!

## Reference

https://ui.shadcn.com/docs/installation/manual

### CODE REVIEW

File: packages/ui/README.md [hunk 1]
```diff
@@ -104,3 +104,243 @@ Modify the `tailwind.config.ts` file to make global style changes to the package
...
+  }
+}
+```
+
+2. Install dependencies
+
+Run the following command from the root of your project:
+
+```shell
+pnpm add tailwindcss-animate class-variance-authority --filter ui
```
Comment (PatrykKuniczak) [inline packages/ui/README.md:143]:
```suggestion
pnpm add tailwindcss-animate class-variance-authority -F ui
```
As you suggest to me prevoiusly :)

File: packages/ui/README.md [hunk 2]
```diff
@@ -104,3 +104,243 @@ Modify the `tailwind.config.ts` file to make global style changes to the package
...
+
+Remember to adjust any paths or package names if your project structure differs from the assumed layout in this guide. 
+
+6. Export components
+
+Edit the `index.ts` file in the `packages/ui` directory to export the button component:
+
+```ts
+// export * from './lib/components'; // remove this line: duplicated button component
+export * from './lib/components/ui/button';
```
Comment (PatrykKuniczak) [inline packages/ui/README.md:345]: IMO better option is create `index.ts` inside `components/ui`, and user after add new component need to add to new import there and main `index.ts` will be untouched.

**Review by PatrykKuniczak — CHANGES_REQUESTED** [review]
Create info or add links in main README to this module and other which include README, because it can be more users friendly.

And now we need 2 identical components named `Button` exported from `ui`
image

And it's sad, they have react components named with lowercase, but we can't deal with it, i think.

Afterall it works for me :)

File: packages/ui/README.md (hunk 2 above)
Comment (Jonghakseo) [reply packages/ui/README.md]: [docs: edit export components section](https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite/pull/739/commits/ba3017e4330527056c1db1ea3d026e7f7ddf19da)

**Pull request merged**